people/compensation.md

```diff
@@ -64,5 +63,58 @@ We have [internal salary documents for the specific positions and levels](https:
 
 ## Benefits
 
-Benefits are offered by {term}`Code for Science and Society`.
-They are offered via [TriNet](https://www.trinet.com/) as described in [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).
+This section describes our benefits policy across all team members at 2i2c. 2i2c team members include people who are designated as employees and as contractors, depending on their role at 2i2c.
+
+Benefits are available to employees only and are managed through {term}`Code for Science and Society`’s relationship with a service partner or PEO. For example, employees who are US based and employed through {term}`Code for Science and Society`’s relationship with [Trinet](https://www.trinet.com/), will have access to health coverage and retirement benefits through Trinet. Employees based outside the US and employed through a relationship with a PEO like Remote will have access to benefits available in their region through that PEO. More details can be found at the [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).
+
+Contractors are independent, not employed by CS&S, and are brought on to support 2i2c with their specific expertise. To support these contract relationships, 2i2c will cover some types of expenses related to the operation of independent contract business, such as would typically be changed by overhead at a larger organization. These expenses include healthcare, accounting services, and reimbursable equipment expenses. We estimate a 20% on top of the contractor monthly payment to cover for these sort of expenses.
+
+### Time off
+
+Our time off policy is documented in our [Taking time off](./time-off.md) section.
+
+### Mechanism for payment
+
+For anything that involves purchasing equipment, services, tickets, etc, we have these options:
+
+* Employee team members can pay via their own methods, and request a reimbursement via Ramp.
```

Comment:
Link to https://compass.2i2c.org/en/latest/administration/reimburse.html as source of truth as for reimbursements for employees it's via invoice not ramp I believe.

Reply:
Updated! Thanks for the reminder!